tests/track/loader_test.py

```diff
@@ -1327,11 +1338,22 @@ def test_post_processes_track_spec(self):
             ],
             "corpora": [
                 {
-                    "name": "unittest",
+                    "name": "unittest-more-than-1k-documents",
```

Comment:
Can we rename this test?

```suggestion
                    "name": "unittest-1k-documents",
```

Reply:
The idea of this test is to check how test mode processor changes the track spec. On entry there's more than 1k documents in this corpus, but after test mode is applied it gets reduced to 1k. What do you think about [e2b2ca0](https://github.com/elastic/rally/pull/1792/commits/e2b2ca084e871f6046285d0c30c5700d5e1098e4) wording?